PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp

```diff
@@ -43,12 +43,6 @@ static HHOOK	hookMouse		= NULL;
 
 static LRESULT CALLBACK hookProcMouse(UINT nCode, WPARAM wParam, LPARAM lParam)
 {
-    if(nCode < 0)

```

Comment:
The correct fix would be to declare `nCode` as `int`.
https://msdn.microsoft.com/en-us/library/windows/desktop/ms644988.aspx

Reply:
And better remove the cast from here:
https://github.com/alexandear/notepad-plus-plus/blob/baf3b3dd018dce526c932d417ae6c1e8805afdd3/PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp#L299

which would avoid the mistake.

Reply:
Indeed, `if(nCode < 0)` is always false.